A.css

```diff
@@ -1,3 +1,3 @@
-h1{
+.title{
```

Comment:
- [Optional] It would be awesome if you change the color property to a different color, as the white title is not visible against the white background on the webpage. 

![image](https://user-images.githubusercontent.com/60618877/176215390-a5395dd1-a969-4e45-98da-b0d0075eced7.png)

It is only visible after selecting the text. 😃